manage_people/tests/test_views.py

```diff
@@ -257,17 +277,17 @@ def test_get_enrollments_added_through_tool(self, mock_get_all_list_data):
         self.assertIn('badge_label_name', result[0])
         self.assertDictContainsSubset(canvas_enrollments[0], result[0])
 
-    def test_registrar_fed_enrollments_not_returned(self, mock_get_all_list_data):
+    def test_registrar_fed_enrollments_not_returned(self, mock_get_all_list_data,
+                                                    mock_role_map):
         """ Ensure that registrar-fed enrollments aren't returned. """
         # set up the mock data
         user_id = uuid.uuid4().hex
         course_instance_id = uuid.uuid4().int
         canvas_enrollments = [  # result of enrollments.list_enrollments_sections
             {'user': {'sis_user_id': user_id, 'sortable_name': 'Burn, Acid'},
-             'role': 'Guest'},
-        ]
-        course_enrollees = [  # result of query for non-feed enrollments
+             'role': 'Guest', 'role_id': 9},
```

Comment:
I see the '}' after the 9, but I don't see the '{' to start. this might be extra or missing.